src/lkl/setup.c

```diff
@@ -1232,6 +1227,7 @@ static int lkl_termination_thread(void* args)
     lthread_exit(NULL);
 
     SGXLKL_VERBOSE("done\n");
```

Comment:
There should be a warning that this debug like is unreachable if the exit call has the correct annotation.

Reply:
Interestingly this didn't happen. I removed the debug line now.